src/drivers/code_hosting/activate.sh

```diff
@@ -14,6 +14,10 @@ function activate_driver_for_code_hosting {
     activate_driver 'code_hosting' 'github'
   elif [ "$origin_hostname" == 'bitbucket.org' ]; then
     activate_driver 'code_hosting' 'bitbucket'
+  elif [ "$(contains_string "$origin_hostname" github)" == true ]; then
+    activate_driver 'code_hosting' 'github'
+  elif [ "$(contains_string "$origin_hostname" bitbucket)" == true ]; then
+    activate_driver 'code_hosting' 'bitbucket'
```

Comment:
It is, in features/git-town-new-pull-request/bitbucket.feature in this PR:

<img width="1846" alt="kg_ssh_identities_by_kevgo_ _pull_request__740_ _originate_git-town" src="https://cloud.githubusercontent.com/assets/268934/19585940/a5f946b8-9705-11e6-89ae-757bb062c6c2.png">